documentation/modules/ROOT/pages/modules/cdc-mysql-connector/c_how-the-mysql-connector-performs-database-snapshots.adoc

```diff
@@ -9,7 +11,7 @@ When your {prodname} MySQL connector is first started, it performs an initial _c
 NOTE: This is the default snapshot mode which is set as `inital` in the `snapshot.mode` property. For other snapshots modes, please check out the xref:assemblies/cdc-mysql-connector/as_deploy-the-mysql-connector.adoc#mysql-connector-configuration-properties_{context}[MySQL connector configuration properties].
 
 ifeval::["{isImageReady}" == "true"]
-image:../../../assets/images/debezium-architecture.png[]
+image:debezium-architecture.png[Debezium Architecture]
```

Comment:
I'm concerned this change could impact the downstream docs in some negative way.  @fbolton can you confirm whether this combined with the `:imagesidr` above will work for you?

Reply:
It's a plain AsciiDoc concept, so I'd *expect* it to be fine. But we can merge it without this change, if you prefer, @Naros.

Reply:
Nah I actually think we can merge it as-is.  I think if downstream it is an issue, they can also send us a PR to change it to as they see fit too.

Reply:
@gunnarmorling @Naros I think this particular image reference might already be broken in the downstream and we just didn't notice it, because the condition hides the image.

One thing I would strongly recommend, however, is to set the `imagesdir` attribute only at the highest possible level (and thus only in one place), e.g. in the https://github.com/debezium/debezium/blob/master/documentation/modules/ROOT/pages/index.adoc file. This is more DRY and it means we can easily change the `imagesdir` to be something different downstream (without having to hunt down and edit multiple files).

Reply:
I agree @fbolton, but I'd probably go one step farther and consider defining these (if possible) in the playbook.yml directly (Antora 2.2 and prior) or antora.yml (Antora 2.3+).

Reply:
To add to the above, right now we have attributes defined in multiple places:

* Individual adoc files
* _attributes.adoc, but requires each page to include it for it to work
* playbook.yml / playbook_author.yml, but requires managing them across multiple files

In Antora 2.3, we can effectively consolidate all these into one single place, https://github.com/debezium/debezium/blob/master/documentation/antora.yml.  For situations where a build-type (e.g. author mode) needs to override an attribute, we can override it from the playbook perspective still, but that should be the only exception.

Reply:
@Naros, so where does that leave us at in terms of this PR? I'm all for consolidating this, but I suppose it should be done separately. Can you capture all this in the DBZ issue you logged for the Antora follow-up work? Is there anything that needs to be done here right now?

Reply:
This one is good to go, I'll add this as part of [DBZ-1687](https://issues.redhat.com/browse/DBZ-1687).

Reply:
Excellent, thanks. Will merge this one here, and added DBZ-1687 to the "1.1-next" bucket.